src/controllers/controller.doughnut.js

```diff
@@ -185,6 +185,18 @@ export default class DoughnutController extends DatasetController {
 		return 0;
 	}
 
+	getLabelAndValue(index) {
+		const me = this;
+		const meta = me._cachedMeta;
+		const chart = me.chart;
+		const labels = chart.data.labels || [];
```

Comment:
I think `meta.rScale.getLabels()` here could make some future changes cleaner.

Reply:
Is that set in the doughnut chart?

Reply:
Looks like its not.